app/src/main/java/com/practicum/playlistmaker/SearchActivity.kt

```diff
@@ -57,36 +89,119 @@ class SearchActivity : AppCompatActivity() {
-            Track("Girl", "The Beatles", "4:11",
-                "https://s3-alpha-sig.figma.com/img/d7b8/cd0b/608881a0d071e280c2c3559fdf879901?Expires=1717372800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=YEIUEWnTfa4KlTjpW4XpjSuX~F57YEXnUDamCPwD7Or4jRaF7AH2wIJCgk1QHnQ1l~9iU~DypUqsyUBqmq0pW~NbBiVTelgDMYDrhptBmJG1WUpwevBLnTEa-xVv5Ey2-cjTvpH38v62Vo5~fYdKdO2ueJarXiTpHrs2ogstL5SqQ1TlAM9lR69Zzw2IlWr18WjuJNDuU-rImPOsfBCfE8jcz~iNQAe6eDQCRSV4xMz9Tk8d29ZwJQJkGwL~dMrQMWfkU03Iy5y1Kd2tst6Cp4eE-JyfbJt9R6Jc7hAtT-kJvUvvXNaF3xDTJ857vQKWJGDBsouRU9hpZwxGgNemrg__"),
-            Track("Michelle", "The Beatles", "3:01",
-                "https://s3-alpha-sig.figma.com/img/d7b8/cd0b/608881a0d071e280c2c3559fdf879901?Expires=1717372800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=YEIUEWnTfa4KlTjpW4XpjSuX~F57YEXnUDamCPwD7Or4jRaF7AH2wIJCgk1QHnQ1l~9iU~DypUqsyUBqmq0pW~NbBiVTelgDMYDrhptBmJG1WUpwevBLnTEa-xVv5Ey2-cjTvpH38v62Vo5~fYdKdO2ueJarXiTpHrs2ogstL5SqQ1TlAM9lR69Zzw2IlWr18WjuJNDuU-rImPOsfBCfE8jcz~iNQAe6eDQCRSV4xMz9Tk8d29ZwJQJkGwL~dMrQMWfkU03Iy5y1Kd2tst6Cp4eE-JyfbJt9R6Jc7hAtT-kJvUvvXNaF3xDTJ857vQKWJGDBsouRU9hpZwxGgNemrg__"),
-            Track("Eleanor Rigby", "The Beatles", "6:12",
-                "https://s3-alpha-sig.figma.com/img/d7b8/cd0b/608881a0d071e280c2c3559fdf879901?Expires=1717372800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=YEIUEWnTfa4KlTjpW4XpjSuX~F57YEXnUDamCPwD7Or4jRaF7AH2wIJCgk1QHnQ1l~9iU~DypUqsyUBqmq0pW~NbBiVTelgDMYDrhptBmJG1WUpwevBLnTEa-xVv5Ey2-cjTvpH38v62Vo5~fYdKdO2ueJarXiTpHrs2ogstL5SqQ1TlAM9lR69Zzw2IlWr18WjuJNDuU-rImPOsfBCfE8jcz~iNQAe6eDQCRSV4xMz9Tk8d29ZwJQJkGwL~dMrQMWfkU03Iy5y1Kd2tst6Cp4eE-JyfbJt9R6Jc7hAtT-kJvUvvXNaF3xDTJ857vQKWJGDBsouRU9hpZwxGgNemrg__"),
-            Track("Come Together", "The Beatles", "4:09",
-                "https://s3-alpha-sig.figma.com/img/d7b8/cd0b/608881a0d071e280c2c3559fdf879901?Expires=1717372800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=YEIUEWnTfa4KlTjpW4XpjSuX~F57YEXnUDamCPwD7Or4jRaF7AH2wIJCgk1QHnQ1l~9iU~DypUqsyUBqmq0pW~NbBiVTelgDMYDrhptBmJG1WUpwevBLnTEa-xVv5Ey2-cjTvpH38v62Vo5~fYdKdO2ueJarXiTpHrs2ogstL5SqQ1TlAM9lR69Zzw2IlWr18WjuJNDuU-rImPOsfBCfE8jcz~iNQAe6eDQCRSV4xMz9Tk8d29ZwJQJkGwL~dMrQMWfkU03Iy5y1Kd2tst6Cp4eE-JyfbJt9R6Jc7hAtT-kJvUvvXNaF3xDTJ857vQKWJGDBsouRU9hpZwxGgNemrg__"),
 
-            )
+        }
+    }
+    private fun search() {
+        if (inputEditText.text.isNotEmpty()) {
+            iTunesService.search(inputEditText.text.toString()).enqueue(object :
+                Callback<TracksResponse> {
+                override fun onResponse(call: Call<TracksResponse>,
+                                        response: Response<TracksResponse>
+                ) {
+                    if (response.code() == 200) {
```

Comment:
```suggestion
                    if (response.isSuccessfull) {
```
Чтобы не хардкодить коды http запросов в активити, можно воспользоваться свойством `isSuccessfull`